.gitignore

```diff
@@ -0,0 +1,398 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
I think you don't need 2 .gitignoree files. You can put everything in the root .gitignore if I am not mistaken

Reply:
still with issues with .gitignore , not working as inteded, testing different alternatives